guac.yaml

```diff
@@ -0,0 +1,85 @@
+package:
+  name: guac
+  version: 0.1.0
+  epoch: 0
+  description: Autoscaling components for Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/guacsec/guac
+      tag: v${{package.version}}
+      expected-commit: 01984331b7337492b833d6b96ddb020d56654e39
+
+  - uses: go/build
```

Comment:
Could we put these in the subpackage pipelines? You could even tidy things up with a `range` if you wanted to.

Reply:
I tried to use a loop/range, but it doesn't appear we support template replacement of range variables inside the inputs to a pipeline :(